snap/squashfs/squashfs_test.go

```diff
@@ -670,6 +671,9 @@ func (s *SquashfsTestSuite) TestBuildSupportsMultipleExcludesWithOnlyOneWildcard
 	})()
 	mksq := testutil.MockCommand(c, "mksquashfs", "")
 	defer mksq.Restore()
+	defer squashfs.MockTruncateSnapToMinSize(func(string, int64) error {
+		return nil
+	})()
```

Comment:
Why do you mock snap truncation here? Unless you want to explicitly avoid testing the truncation code in this function, I think you could let it run?

Reply:
Since `mksquashfs` is mocked, there is no snap file to truncate. I could generate one with the mocked `mksquashfs` command, if you think that'd be better?

Reply:
I wonder if SquashfsTestSuite should simply have this `squashfs.MockGrowSnapToMinSiz()` as part of `SetUpTest`?